ui/src/app/views/page-template-signed-out/component.ts

```diff
@@ -0,0 +1,17 @@
+import {Component, Input} from '@angular/core';
+
+@Component({
+  selector: 'app-page-template-signed-out',
+  templateUrl: './component.html',
+  styleUrls: ['./component.css']
+})
+export class PageTemplateSignedOutComponent {
+  @Input() imageSrc: string;
+  headerImg: string;
+
+  constructor() {
+    this.imageSrc = '';
+    this.headerImg = '/assets/images/AoU-logo-registration_nonSignedIn.png';
```

Comment:
This may seem picky but this file name is a mixture of camel-, pascal-, kebab-, _and_ snake-case, could we normalize to one of those conventions?  I don't really care which one so long as its just one and we stick with it for other asset files.

Reply:
Will take care of this

Reply:
I would personally just say kebab- case. I think that will probably be easiest? (And we can probably remove the AoU prefix.)